llvm/lib/TargetParser/TargetParser.cpp

```diff
@@ -323,43 +323,59 @@ void AMDGPU::fillAMDGPUFeatureMap(StringRef GPU, const Triple &T,
                                   StringMap<bool> &Features) {
   // XXX - What does the member GPU mean if device name string passed here?
   if (T.isSPIRV() && T.getOS() == Triple::OSType::AMDHSA) {
-    // AMDGCN SPIRV must support the union of all AMDGCN features.
+    // AMDGCN SPIRV must support the union of all AMDGCN features. This list
```

Comment:
I wonder in the long term whether we want to table gen this.

Reply:
Technically we added those features via `.td` files so it could potentially generate a file for this one to include.